### DISCUSSION TITLE
MCI is blocking REALITY servers in Iran

### DISCUSSION THREAD

**Discussion by WatchDogsDev** [XTLS/Xray-core#2450] [Q&A]
Hi, First thanks for your great work.
I had servers with clean IP using the recommended latest protocol **VLESS+TCP+RPRX-VISION+REALITY** for 2 month and everything was fine.
Recently (about 4 days ago) the biggest internet operator in Iran known as **MCI** started blocking reality server IPs **under 24 hours.**
I have used many less known server providers to get clean IP but it got blocked under 24 hours with around 70 users. 

Any ideas how we can handle this? Or how is it detected ?

**Comment by mkhib** [comment]
Does anyone have any solution to prevent MCI detecting reality servers?

**Reply by Xyncgas** [reply]
Try other protocols and pay attentions to https://foci.community

**Comment by ghost** [comment]
> around 70 users

It could be the number of users or their traffic patterns made it look like a proxy server. Does the same blocking happen with a proxy with only 2 or 3 users?

**Reply by Xyncgas** [reply]
Statistic footprints

**Reply by hawshemi** [reply]
Can you elaborate on that?
What do you mean by "Statistic footprints"?

**Reply by ghost** [reply]
A systematic test #2778 suggests that traffic volume is one factor that triggers blocking.

* VPS with 20 users, heavy traffic, gets disconnected and blocked immediately
* VPS with 5-7 users, low to medium traffic, survives for days, then increased traffic triggers blocking
* VPS with 1-2 users, very low traffic, survives for weeks, then increased traffic triggers blocking

**Reply by Xyncgas** [reply]
Typical communist country's social atomization strategy

Human doing things together achieve greater social impacts, therefore it's banned, you can visit unknown server by yourself that means you have to evade censorship by yourself which barring people who aren't technological literate from evading the government's technologic censorship through its dominance for it

Such is the result of Iranian allowing themselves be shepherded into this situation

The people in other countries believes in social atomism to some extent too, employing surveillance to get to know you so to do a lot things most efficiently, sometimes selling you products or maybe tricking you or forcing you to do things in more extreme cases like china and their social credit system as seen in 2019-2022

We're working for god to develop the technology to fight back, because we are supposed to see to what's right with the dictates of our own conscience instead of someone else's eyes and words to see it to tell it for us and we should act out of freewill out of our own morals to be united under consent to make our lives meaningful to embrace a democratic society over one person's tyranny
